docs/adr/014-rollover-2.0.md

```diff
@@ -0,0 +1,66 @@
+# ADR 014 - Rollover 2.0
+
+```
+Author: @thomaseizinger
+Date: 20/01/2022
+Status: Active
+```
+
+## Summary
+
+Revise the design of the rollover protocol to:
+
+1. Run at fixed points in time.
+2. Not require approval by either party.
+
+## Context
+
+Currently, rollover is a protocol that is initiated by the taker pretty much on the hour every hour.
+The exact conditions for triggering are that we want the timestamp of the price event that is used for attestation to be 24h in the future.
+Because the oracle attests on the price exactly every hour, we can rollover to a new price event on the hour.
+
+Rollover is coupled to paying fees.
+In particular, the time window until the attestation can be considered as the current funding period.
```

Comment:
So far our funding interval was one hour, i.e. we charge every hour. That is not the same as the settlement interval that was 24h so far.

Are we planning to align them, so that the settlement interval is always the same as the funding interval, i.e. if the next funding event is at `04:00 UTC` then the oracle event ID is also at `04:00 UTC`? 
We cannot rollover exactly at the time of the oracle event ID, because once attested we cannot rollover anymore but have to force close. There needs to be an agreement between the two parties to roll-over before the expiry timestamp. 

Would it make sense to hard-code that rollover is always triggered e.g. 5 minutes before the expiry event?

Reply:
> Are we planning to align them, so that the settlement interval is always the same as the funding interval, i.e. if the next funding event is at `04:00 UTC` then the oracle event ID is also at `04:00 UTC`?

Yes, I think we should do that.

> because once attested we cannot rollover anymore but have to force close.

That is the policy we currently have but I don't think we'd have to be that strict here. We could delay force-closing for a couple of minutes.

> There needs to be an agreement between the two parties to roll-over before the expiry timestamp.

Yep. I don't yet know, what the best solution is here.